docs/additional-resources/policies.md

```diff
@@ -10,6 +10,18 @@ For users requiring secure research computing environments to ensure compliance
 
 <br>
 
+## Core storage data retention
+
+The data retention policy for CU Research Computing “core” storage (user data in /home and /projects spaces) is a period of six years beyond the user’s last login to a CU Research Computing service. This duration is in alignment with the University’s data retention policy for research records associated with grants and contracts. The policy will be enacted via a “5 + 1” approach: after a period of five years since the user’s last login, a communication will be sent to the email address associated with the user’s account, and cc’d (as a courtesy) to persons belonging to the group associated with the user’s /projects space. The communication will note that data in their /home and /projects spaces will be archived after one additional month and purged after one additional year if no action is taken by the user or group members to retain the data. The user will be given the opportunity to request that the data not be archived if it is still needed on CURC systems, or to copy the data from the archive to an alternate offsite location during the final year.  If the user (and only the user) does not respond within the year, the data will be deleted upon reaching year six. 
+
```

Comment:
One note I have on the above passage 
```
The user will be given the opportunity to request that the data not be archived if it is still needed on CURC systems, or to copy the data from the archive to an alternate offsite location during the final year.
```
I am fine with it as stated, but I think it could be misinterpreted. Particularly, someone may assume that "to request that the data not be archived" means that it will persist beyond the final year.

Reply:
I don't think this is our intention, so I thought I would bring it up.

Reply:
Updated per conversation in slack